NEWS.md

```diff
@@ -1,6 +1,15 @@
 erGW - 3GPP GGSN and PDN-GW in Erlang
 =====================================
 
+Version 2.5.0 - 04 August 2020
+----------------------------
+* implement path monitoring profiles for outgoing (active) paths
+* bump ergw_aaa dependency to 3.5.0 to support  
+  [state stats](https://github.com/travelping/ergw_aaa/blob/master/doc/aaa_session_metrics.md) and
+  diameter [avp filters](https://github.com/travelping/ergw_aaa/blob/master/doc/diameter_avp_filter.md)
```

Comment:
can you change the links to point to the actual tag, e.g.:

https://github.com/travelping/ergw_aaa/blob/3.5.0/doc/diameter_avp_filter.md